config.py

```diff
@@ -71,6 +71,11 @@ class Config(object):
     DM_APP_NAME = 'buyer-frontend'
     DM_DOWNSTREAM_REQUEST_ID_HEADER = 'X-Amz-Cf-Id'
 
+    DM_FRAMEWORK_CONTENT_MAP = {
```

Comment:
Yeah - I needed something that could be shared between the loading up of the content (i.e. into the app on startup), and the getting of it (e.g. in the service page view). Hard-coding a bunch of stuff about G4/5/6 in both places seemed like I hadn't really made much progress. But I totally dig that this is really still the same thing.

On the second point, I don't know if there's a memory impact of loading up 5 copies of the same content (G4/5/6/7/8) but it sure does seem a bit wrong :-D